pkgs/os-specific/linux/ifmetric/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, lynx }:
+
+stdenv.mkDerivation rec {
+  pname = "ifmetric";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "http://0pointer.de/lennart/projects/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "1v0s5x81jzwnnl7hr254d4nkyc8qcv983pzr6vqmbr9l9q553a0g";
+  };
+
+  patches = [
+    (fetchurl {
```

please add a comment about what this patch is for, and when to remove it (if relevant)
```suggestion
  patches = [
     # does X, Y, Z. Can be removed (next patch | never)
    (fetchurl {
```